features/gift_exchanges/challenge_giftexchange.feature

```diff
@@ -219,8 +215,6 @@ Feature: Gift Exchange Challenge
       And I should see "Regenerate All Potential Matches"
       And I should see "try regenerating assignments"
     When I follow "Regenerate Assignments"
-      And the system processes jobs
-      And I wait 3 seconds
       And I reload the page
```

Comment:
I was unsure whether to remove the steps that "reload the page" because it was unclear whether they were related or not. please advise.

Reply:
Yup, you'll still need reload the page here. When you press Regenerate Assignments, it takes you back to /potential_matches, which just shows a page like:

<img width="1080" alt="AO3 page explaining assignments are being generated." src="https://user-images.githubusercontent.com/907055/228414720-29f3a8d4-9452-41f7-bc86-5c1e38840f8c.png">

If you don't reload, you won't see the regenerated assignments.